Quasardb.Native/qdb_sized_string.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+// ReSharper disable BuiltInTypeReferenceStyle
+// ReSharper disable InconsistentNaming
+
+using qdb_size_t = System.UIntPtr;
+
+namespace Quasardb.Native
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public unsafe struct qdb_sized_string
+    {
+        public Char* data;
+        public qdb_size_t length;
+
+        public override string ToString()
+        {
+            if (length.ToUInt64() == 0UL) return string.Empty;
+            // TODO: limited to 32-bit
```

Comment:
Is it problematic to be limited to 32-bit, is that used to other things than aliases (which are limited in size anyway)?

Reply:
This is used just for column names and query error message.

Note however that blobs are limited to 32-bit too (that's what this code was based on).

Reply:
Ok.